### PR TITLE
docs: fix typos and clarify a few things on the ide page

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

**Comment by romanowski** [comment]
Thank you @ckipp01 for this contribution!  I rebased the PR and change wording in a few places.

File: website/docs/guides/ide.md [hunk 1]
```diff
@@ -32,26 +34,46 @@ Scala CLI was tested with two main IDEs for Scala:
...
 
-In ideal world, we would replace the rest of this Guide with something along the lines of: `Scala CLI works within IDEs above as you would expect` however mainly due to how fresh Scala CLI is and also due to our radical approach to the project structure using Scala CLI -powered project to your favourite IDE may not be as amazing as we would like.
+In ideal world, we would replace the rest of this Guide with something along the
+lines of: `Scala CLI works with all IDEs above as you would expect` however
+mainly due to how fresh Scala CLI is and also due to our radical approach to the
+project structure using a Scala CLI-powered projects with your favourite IDE may
+not be as amazing as we would like to be.
 
-Proper IDE integration is our top priority at this moment.
+Proper IDE integration is our top priority at them moment.
```
Comment (ckipp01) [inline website/docs/guides/ide.md]:
```suggestion
Proper IDE integration is our top priority at the moment.
```

Comment (ckipp01) [reply website/docs/guides/ide.md]: Sorry mean this to be the not them.

File: website/docs/guides/ide.md [hunk 2]
```diff
@@ -32,26 +34,46 @@ Scala CLI was tested with two main IDEs for Scala:
...
-Generally, mainly due to problems with keeping project structure up to date Metals can assists with writing code using Scala CLI, however it cannot be the source of true and we recommend falling back to command line in such cases.
+Generally, mainly due to problems with keeping project structure up to date
+Metals can assists with writing code while using Scala CLI, but it cannot be the
+source of truth and we recommend falling back to command line in such cases.
 
 ## IntelliJ
 
 In terms of IntelliJ, the most significant problem is that IntelliJ requires sources to be placed within a directory. This means that top level sources are not recognized as proper sources.
 
+<<<<<<< HEAD
```
Comment (ckipp01) [inline website/docs/guides/ide.md]: Also there is still git conflicts in this page

**Pull request merged**